### ISSUE TITLE
Add tests for `flame_texturepacker`

### ISSUE THREAD

**Issue by spydon** [flame-engine/flame#2989]
# What could be improved
Add tests for `flame_texturepacker`.

# Why should this be improved
Because we should have a well tested codebase.

# Any risks?
No.

# More information
https://github.com/flame-engine/flame/tree/main/packages/flame_texturepacker

**Comment by ycfay** [comment]
i use the flame_texturepacker , Paired with SpriteAnimationGroupComponent， but can't work.

the code 
`
  @override
  Future<void> prepareAnimations() async {
    for (var action in PlayerState.values) {
      animations = {};
      final sprites = atlas.findSpritesByName(action.name);
      if (sprites.isNotEmpty) {
        SpriteAnimation animation = SpriteAnimation.spriteList(
          sprites,
          stepTime: 0.2,
          loop: true,
        );
        animations!.addAll({action: animation});
      }
    }
    current = PlayerState.idle_down;
    size = Vector2.all(60);
    //         add(
    //   SpriteAnimationComponent(scale: Vector2.all(0.5), animation: animation),
    // );
  }
`

help plz！

**Comment by ycfay** [comment]
i use the flame_texturepacker , Paired with SpriteAnimationGroupComponent， but can't work.

the code 
`
  @override
  Future<void> prepareAnimations() async {
    for (var action in PlayerState.values) {
      animations = {};
      final sprites = atlas.findSpritesByName(action.name);
      if (sprites.isNotEmpty) {
        SpriteAnimation animation = SpriteAnimation.spriteList(
          sprites,
          stepTime: 0.2,
          loop: true,
        );
        animations!.addAll({action: animation});
      }
    }
    current = PlayerState.idle_down;
    size = Vector2.all(60);
    //         add(
    //   SpriteAnimationComponent(scale: Vector2.all(0.5), animation: animation),
    // );
  }
`

help plz！

**Comment by spydon** [comment]
@ycfay this is not related to the issue you posted in, please open a new one (and make sure to format your code properly).

**Comment by ryzizub** [comment]
@spydon im looking into this one, you can assign it to me :) gonna raise it too 100 💪

**Comment by spydon** [comment]
Awesome @ryzizub! :blue_heart:

**Comment by ryzizub** [comment]
@spydon im going to tackle this in hacktoberfest can I ask for adding a tag ❤️ ?

**Comment by spydon** [comment]
> @spydon im going to tackle this in hacktoberfest can I ask for adding a tag ❤️ ?

The tag is already on the whole repo which is enough, but I'll add the tag here too! :)

**Comment by spydon** [comment]
@ryzizub not hacktober anymore, but are you still planning to tackle this? :innocent:

**Comment by ryzizub** [comment]
@spydon yep, had 0 time in October and until this month. During holidays till end of the year I should tackle that 😇
